2025-03-10 12:34:24,756 [INFO] USER:chintamani_chourase handlers.py:106:syndicate LOG: Configuration used: C:\Users\chintamani_chourase\IdeaProjects\aws-lamda\.syndicate-config-dev
2025-03-10 12:34:27,012 [WARNING] USER:chintamani_chourase __init__.py:190:initialize_project_state LOG: [93mConfig is set and generated but project state does not exist, seems that you've come from the previous version.[0m
2025-03-10 12:34:27,013 [WARNING] USER:chintamani_chourase __init__.py:193:initialize_project_state LOG: [93mGenerating project state file (.syndicate) from the existing structure...[0m
2025-03-10 12:34:27,033 [INFO] USER:chintamani_chourase processor.py:143:_prepare_bucket_view LOG: Viewing complement, URIBucketView, has been found, setting up the raw data.
2025-03-10 12:34:28,524 [INFO] USER:chintamani_chourase project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-10 12:34:28,524 [INFO] USER:chintamani_chourase project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-10 12:34:28,934 [INFO] USER:chintamani_chourase sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-10 12:34:28,935 [INFO] USER:chintamani_chourase helper.py:658:check_lambda_name LOG: Validating lambda name: 'hello_world'
2025-03-10 12:34:28,936 [INFO] USER:chintamani_chourase helper.py:673:check_lambda_name LOG: Lambda name: 'hello_world' passed the validation
2025-03-10 12:34:28,936 [INFO] USER:chintamani_chourase helper.py:647:resolve_project_path LOG: Parameter: 'project_path' wasn't specified. Getting automatically
2025-03-10 12:34:28,936 [INFO] USER:chintamani_chourase helper.py:651:resolve_project_path LOG: Path: 'C:/Users/chintamani_chourase/IdeaProjects/aws-lamda' was assigned to the parameter: 'project_path'
2025-03-10 12:34:28,937 [INFO] USER:chintamani_chourase generate.py:116:lambda_function LOG: Lambda names: ('hello_world',)
2025-03-10 12:34:28,938 [INFO] USER:chintamani_chourase generate.py:117:lambda_function LOG: Runtime: nodejs
2025-03-10 12:34:28,938 [INFO] USER:chintamani_chourase generate.py:118:lambda_function LOG: Project path: C:/Users/chintamani_chourase/IdeaProjects/aws-lamda
2025-03-10 12:34:28,954 [INFO] USER:chintamani_chourase lambda_function.py:464:_generate_nodejs_lambdas LOG: Lambda hello_world created
2025-03-10 12:34:28,956 [INFO] USER:chintamani_chourase lambda_function.py:169:generate_lambda_function LOG: Lambda hello_world has been successfully added to the project.
2025-03-10 12:34:28,957 [INFO] USER:chintamani_chourase helper.py:330:timed LOG: Stage lambda_function, elapsed time: 0:00:00.019632
2025-03-10 12:35:31,286 [INFO] USER:chintamani_chourase handlers.py:106:syndicate LOG: Configuration used: C:\Users\chintamani_chourase\IdeaProjects\aws-lamda\.syndicate-config-dev
2025-03-10 12:35:33,534 [INFO] USER:chintamani_chourase processor.py:143:_prepare_bucket_view LOG: Viewing complement, URIBucketView, has been found, setting up the raw data.
2025-03-10 12:35:34,690 [INFO] USER:chintamani_chourase project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-10 12:35:34,691 [INFO] USER:chintamani_chourase project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-10 12:35:35,085 [INFO] USER:chintamani_chourase sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-10 12:35:35,086 [INFO] USER:chintamani_chourase helper.py:658:check_lambda_name LOG: Validating lambda name: 'hello_world'
2025-03-10 12:35:35,086 [INFO] USER:chintamani_chourase helper.py:673:check_lambda_name LOG: Lambda name: 'hello_world' passed the validation
2025-03-10 12:35:35,086 [INFO] USER:chintamani_chourase helper.py:647:resolve_project_path LOG: Parameter: 'project_path' wasn't specified. Getting automatically
2025-03-10 12:35:35,087 [INFO] USER:chintamani_chourase helper.py:651:resolve_project_path LOG: Path: 'C:/Users/chintamani_chourase/IdeaProjects/aws-lamda' was assigned to the parameter: 'project_path'
2025-03-10 12:35:35,088 [INFO] USER:chintamani_chourase generate.py:116:lambda_function LOG: Lambda names: ('hello_world',)
2025-03-10 12:35:35,088 [INFO] USER:chintamani_chourase generate.py:117:lambda_function LOG: Runtime: nodejs
2025-03-10 12:35:35,088 [INFO] USER:chintamani_chourase generate.py:118:lambda_function LOG: Project path: C:/Users/chintamani_chourase/IdeaProjects/aws-lamda
2025-03-10 12:35:35,115 [INFO] USER:chintamani_chourase lambda_function.py:464:_generate_nodejs_lambdas LOG: Lambda hello_world created
2025-03-10 12:35:35,116 [INFO] USER:chintamani_chourase lambda_function.py:169:generate_lambda_function LOG: Lambda hello_world has been successfully added to the project.
2025-03-10 12:35:35,120 [INFO] USER:chintamani_chourase helper.py:330:timed LOG: Stage lambda_function, elapsed time: 0:00:00.032504
2025-03-10 13:01:50,554 [INFO] USER:chintamani_chourase handlers.py:106:syndicate LOG: Configuration used: C:\Users\chintamani_chourase\IdeaProjects\aws-lamda\.syndicate-config-dev
2025-03-10 13:01:52,811 [INFO] USER:chintamani_chourase processor.py:143:_prepare_bucket_view LOG: Viewing complement, URIBucketView, has been found, setting up the raw data.
2025-03-10 13:01:54,110 [INFO] USER:chintamani_chourase project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-10 13:01:54,110 [INFO] USER:chintamani_chourase project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-10 13:01:54,521 [INFO] USER:chintamani_chourase sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-10 13:01:55,911 [INFO] USER:chintamani_chourase handlers.py:142:test LOG: Running tests...
2025-03-10 13:01:55,912 [INFO] USER:chintamani_chourase handlers.py:150:test LOG: Tests not found, 'tests' folder is missing in 'C:/Users/chintamani_chourase/IdeaProjects/aws-lamda'.
2025-03-10 13:01:55,913 [INFO] USER:chintamani_chourase helper.py:330:timed LOG: Stage test, elapsed time: 0:00:00.001278
2025-03-10 13:01:55,949 [INFO] USER:chintamani_chourase handlers.py:993:assemble LOG: Building artifacts, bundle: aws-lamda_250310.130154
2025-03-10 13:01:55,950 [INFO] USER:chintamani_chourase project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\chintamani_chourase\IdeaProjects\aws-lamda\jsrc\main\java\com\awslamda.
2025-03-10 13:01:55,951 [INFO] USER:chintamani_chourase project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\chintamani_chourase\IdeaProjects\aws-lamda\src\lambdas.
2025-03-10 13:01:55,952 [INFO] USER:chintamani_chourase project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\chintamani_chourase\IdeaProjects\aws-lamda\app\lambdas.
2025-03-10 13:01:55,952 [INFO] USER:chintamani_chourase project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\chintamani_chourase\IdeaProjects\aws-lamda\dnapp\lambdas.
2025-03-10 13:01:55,952 [INFO] USER:chintamani_chourase project_state.py:430:_update_bpm_resources_from_path LOG: Going to resolve any build project mapping dependent resources from a given path: C:\Users\chintamani_chourase\IdeaProjects\aws-lamda\swagger_src.
2025-03-10 13:01:55,952 [INFO] USER:chintamani_chourase project_state.py:430:_update_bpm_resources_from_path LOG: Going to resolve any build project mapping dependent resources from a given path: C:\Users\chintamani_chourase\IdeaProjects\aws-lamda\appsync_src.
2025-03-10 13:01:55,952 [INFO] USER:chintamani_chourase handlers.py:821:assemble_node LOG: Command assemble node: project_path: app 
2025-03-10 13:01:55,954 [INFO] USER:chintamani_chourase nodejs.py:58:assemble_node_lambdas LOG: Going to package lambdas starting by path C:\Users\chintamani_chourase\IdeaProjects\aws-lamda\app
2025-03-10 13:01:55,955 [INFO] USER:chintamani_chourase handlers.py:830:assemble_node LOG: NodeJS artifacts were prepared successfully.
2025-03-10 13:01:55,955 [INFO] USER:chintamani_chourase helper.py:330:timed LOG: Stage assemble_node, elapsed time: 0:00:00.002727
2025-03-10 13:01:55,957 [INFO] USER:chintamani_chourase helper.py:330:timed LOG: Stage assemble, elapsed time: 0:00:00.008148
2025-03-10 13:01:55,959 [INFO] USER:chintamani_chourase handlers.py:1037:package_meta LOG: Package meta, bundle: aws-lamda_250310.130154
2025-03-10 13:01:55,960 [INFO] USER:chintamani_chourase meta_processor.py:513:create_meta LOG: Bundle path: C:/Users/chintamani_chourase/IdeaProjects/aws-lamda/bundles/aws-lamda_250310.130154
2025-03-10 13:01:55,973 [INFO] USER:chintamani_chourase helper.py:292:write_content_to_file LOG: C:\Users\chintamani_chourase\IdeaProjects\aws-lamda\.syndicate-config-dev/bundles/aws-lamda_250310.130154\build_meta.json file was created.
2025-03-10 13:01:55,973 [INFO] USER:chintamani_chourase handlers.py:1040:package_meta LOG: Meta was configured successfully.
2025-03-10 13:01:55,973 [INFO] USER:chintamani_chourase helper.py:330:timed LOG: Stage package_meta, elapsed time: 0:00:00.014032
2025-03-10 13:01:56,164 [INFO] USER:chintamani_chourase handlers.py:1083:upload LOG: Upload bundle: aws-lamda_250310.130154
2025-03-10 13:01:56,510 [INFO] USER:chintamani_chourase bundle_processor.py:232:upload_bundle_to_s3 LOG: Bundle was validated successfully
2025-03-10 13:01:56,981 [INFO] USER:chintamani_chourase handlers.py:1090:upload LOG: Bundle was uploaded successfully
2025-03-10 13:01:56,981 [INFO] USER:chintamani_chourase helper.py:330:timed LOG: Stage upload, elapsed time: 0:00:01.004946
2025-03-10 13:01:56,986 [INFO] USER:chintamani_chourase helper.py:330:timed LOG: Stage build, elapsed time: 0:00:02.447624
2025-03-10 13:04:12,982 [INFO] USER:chintamani_chourase handlers.py:106:syndicate LOG: Configuration used: C:\Users\chintamani_chourase\IdeaProjects\aws-lamda\.syndicate-config-dev
2025-03-10 13:04:15,213 [INFO] USER:chintamani_chourase processor.py:143:_prepare_bucket_view LOG: Viewing complement, URIBucketView, has been found, setting up the raw data.
2025-03-10 13:04:16,655 [INFO] USER:chintamani_chourase project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-10 13:04:16,655 [INFO] USER:chintamani_chourase project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-10 13:04:17,063 [INFO] USER:chintamani_chourase sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-10 13:04:17,479 [INFO] USER:chintamani_chourase project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-10 13:04:17,479 [INFO] USER:chintamani_chourase project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-10 13:04:17,885 [INFO] USER:chintamani_chourase sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-10 13:04:17,887 [INFO] USER:chintamani_chourase helper.py:233:resolve_default_value LOG: Resolved value of deploy_name: aws-lamda
2025-03-10 13:04:18,265 [INFO] USER:chintamani_chourase project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-10 13:04:18,265 [INFO] USER:chintamani_chourase project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-10 13:04:18,701 [INFO] USER:chintamani_chourase sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-10 13:04:18,701 [INFO] USER:chintamani_chourase helper.py:233:resolve_default_value LOG: Resolved value of bundle_name: aws-lamda_250310.130154
2025-03-10 13:04:19,116 [INFO] USER:chintamani_chourase project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-10 13:04:19,117 [INFO] USER:chintamani_chourase project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-10 13:04:19,521 [INFO] USER:chintamani_chourase sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-10 13:04:19,940 [INFO] USER:chintamani_chourase project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-10 13:04:19,940 [INFO] USER:chintamani_chourase project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-10 13:04:20,341 [INFO] USER:chintamani_chourase sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-10 13:04:22,698 [INFO] USER:chintamani_chourase deployment_processor.py:466:create_deployment_resources LOG: Prefixes and suffixes of any resource names have been resolved
2025-03-10 13:04:22,698 [INFO] USER:chintamani_chourase deployment_processor.py:489:create_deployment_resources LOG: Going to deploy AWS resources
2025-03-10 13:04:23,333 [INFO] USER:chintamani_chourase deployment_processor.py:74:_process_resources LOG: Processing iam_policy resources
2025-03-10 13:04:25,665 [INFO] USER:chintamani_chourase iam_resource.py:120:_create_policy_from_meta LOG: Created IAM policy cmtr-cdfe3aed-lambda-basic-execution.
2025-03-10 13:04:25,969 [INFO] USER:chintamani_chourase deployment_processor.py:74:_process_resources LOG: Processing iam_role resources
2025-03-10 13:04:33,986 [INFO] USER:chintamani_chourase iam_resource.py:183:_create_role_from_meta LOG: Created IAM role cmtr-cdfe3aed-hello_world-role.
2025-03-10 13:04:33,987 [INFO] USER:chintamani_chourase deployment_processor.py:88:_process_resources LOG: Processing lambda resources
2025-03-10 13:04:35,649 [ERROR] USER:chintamani_chourase deployment_processor.py:387:process_response LOG: Caused by resource named cmtr-cdfe3aed-hello_world. Error while creating lambda: cmtr-cdfe3aed-hello_world;Deployment package cdfe3aed/task01/aws-lamda_250310.130154/hello_world-1.0.zip does not exist in syndicate-education-platform-custom-sandbox-artifacts-sbox01 bucket
2025-03-10 13:04:37,254 [INFO] USER:chintamani_chourase deployment_processor.py:527:create_deployment_resources LOG: Going to apply post deployment tags
2025-03-10 13:04:37,281 [INFO] USER:chintamani_chourase group_tagging_api_resource.py:135:apply_tags LOG: Tags were successfully applied
2025-03-10 13:04:37,281 [INFO] USER:chintamani_chourase deployment_processor.py:530:create_deployment_resources LOG: Going to create deploy output
2025-03-10 13:04:39,181 [INFO] USER:chintamani_chourase bundle_processor.py:71:create_deploy_output LOG: Output file with name aws-lamda_250310.130154/outputs/aws-lamda_failed.json has been created
2025-03-10 13:04:39,181 [INFO] USER:chintamani_chourase deployment_processor.py:560:create_deployment_resources LOG: Deploy output for aws-lamda was created.
2025-03-10 13:04:39,182 [WARNING] USER:chintamani_chourase handlers.py:344:deploy LOG: Backend resources were deployed with errors. See deploy output file.
2025-03-10 13:04:39,182 [INFO] USER:chintamani_chourase helper.py:330:timed LOG: Stage deploy, elapsed time: 0:00:18.840488
2025-03-10 13:04:39,608 [INFO] USER:chintamani_chourase project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-10 13:04:39,608 [INFO] USER:chintamani_chourase project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-10 13:04:39,985 [INFO] USER:chintamani_chourase sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
